File: Difficulty: Easy/Remove Duplicates Sorted Array/remove-duplicates-sorted-array.py
class Solution:
    def removeDuplicates(self, arr):
        j = 0
        uni = arr[0]
        for i in arr:
            if i != uni:
                arr[j] = uni
                j +=1
                uni = i
        arr[j] = i
        return arr[0:j+1:1]
                
            

File: Difficulty: Easy/Remove Duplicates Sorted Array/remove-duplicates-sorted-array.cpp
class Solution {
  public:
    vector<int> removeDuplicates(vector<int> &arr) {
        // code here
        int j=1;
       int n=arr.size();
        for(int i=1;i<n;i++)
        {
            if(arr[i-1]!=arr[i])
            {
                arr[j]=arr[i];
                j++;
            }
        }
        arr.resize(j);
        return arr;
    }
};